<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>AddTaskWidget</class>
 <widget class="QDialog" name="AddTaskWidget">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>600</width>
    <height>150</height>
   </rect>
  </property>
  <property name="minimumSize">
   <size>
    <width>600</width>
    <height>150</height>
   </size>
  </property>
  <property name="maximumSize">
   <size>
    <width>16777215</width>
    <height>150</height>
   </size>
  </property>
  <property name="windowTitle">
   <string>Новый таск</string>
  </property>
  <layout class="QGridLayout" name="gridLayout">
   <item row="0" column="0">
    <widget class="QLabel" name="nameLabel">
     <property name="font">
      <font>
       <family>GOST type B</family>
       <pointsize>12</pointsize>
      </font>
     </property>
     <property name="text">
      <string>Название</string>
     </property>
    </widget>
   </item>
   <item row="0" column="1">
    <widget class="QLineEdit" name="nameLineEdit">
     <property name="font">
      <font>
       <family>GOST type B</family>
       <pointsize>12</pointsize>
      </font>
     </property>
    </widget>
   </item>
   <item row="1" column="0">
    <widget class="QLabel" name="estimatedHoursLabel">
     <property name="font">
      <font>
       <family>GOST type B</family>
       <pointsize>12</pointsize>
      </font>
     </property>
     <property name="text">
      <string>Прим. часов на решение</string>
     </property>
    </widget>
   </item>
   <item row="1" column="1">
    <widget class="QSpinBox" name="estimatedHoursSpinBox">
     <property name="maximumSize">
      <size>
       <width>100</width>
       <height>16777215</height>
      </size>
     </property>
     <property name="font">
      <font>
       <family>GOST type B</family>
       <pointsize>12</pointsize>
      </font>
     </property>
     <property name="maximum">
      <number>100</number>
     </property>
     <property name="singleStep">
      <number>1</number>
     </property>
    </widget>
   </item>
   <item row="2" column="0">
    <widget class="QPushButton" name="createPushButton">
     <property name="font">
      <font>
       <family>GOST type B</family>
       <pointsize>10</pointsize>
      </font>
     </property>
     <property name="text">
      <string>Создать</string>
     </property>
    </widget>
   </item>
  </layout>
 </widget>
 <resources/>
 <connections/>
</ui>
